lawnchair/src/app/lawnchair/ui/preferences/IconPackPreferences.kt

```diff
@@ -156,12 +155,48 @@ fun IconPackPreferences() {
                     onValueChange = {
                         themedIconsAdapter.onChange(newValue = it.themedIcons)
                         drawerThemedIconsAdapter.onChange(newValue = it.drawerThemedIcons)
+                        iconPackAdapter.onChange(newValue = "")
+                        themedIconPackAdapter.onChange(newValue = "")
                     },
                     description = if (themedIconsAvailable.not()) {
                         stringResource(id = R.string.lawnicons_not_installed_description)
                     } else null,
                 )
             }
+
+            ExpandAndShrink(visible = themedIconsAdapter.state.value && !drawerThemedIconsAdapter.state.value) {
```

Comment:
Move `drawerThemedIconsAdapter.state.value` to a new variable, like `drawerThemedIconsEnabled`, to shorten the variable name.